packages/hash/frontend/src/blocks/page/PageBlock.tsx

```diff
@@ -16,6 +16,11 @@ import { BlocksMap, createEditorView } from "./createEditorView";
 import { usePortals } from "./usePortals";
 import { useReadonlyMode } from "../../shared/readonly-mode";
 import { usePageContext } from "./PageContext";
+import {
+  PAGE_CONTENT_WIDTH,
+  PAGE_HORIZONTAL_PADDING_FORMULA,
+  PAGE_MIN_PADDING,
+} from "../../pages/[account-slug]/[page-slug].page";
```

Comment:
Page files are roots of the dependency graph. Importing from these files creates counter-flows and may lead to circular dependencies. Circular dependencies are a source of confusion and pretty tough bugs. I suggest refactoring this diff to avoid imports from `*.page`.

Reply:
Makes sense, thanks for the suggestion, gonna refactor it!